server/views/admin-add-article.hbs

```diff
@@ -1,24 +1,20 @@
 <form method="post" >
     <div class="form-group has-success">
-        <h5 class="text-success" >Title:</h5>
-        <input  class="form-control" id="inputSuccess1" name="title"/>
-    </div>
-    <div class="form-group has-success">
-        <h4> Categories:</h4>
-        <select name="category" class="form-control">
+        <h5 class="text-success mt-5" >Title:</h5>
+        <input  class="col-lg-4 col-md-8 mb-3" id="inputSuccess1" name="title"/>
+         <h5 class="text-success mt-2"> Categories:</h5>
```

Comment:
don't use col-lg- and col-md... to set the size of the forms. Just use form-groups and if you want a different container size then it should be in the parent (but not need for it here)